applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
is `make apis` failing to include `"/v2/about"` ? do we know the reason ? are there more endpoints in the same condition ?
thanks